### ISSUE TITLE
Error when print tokens

### ISSUE THREAD

**Issue by lucasndi** [jay0lee/GAM#306]
Hi guys, i'm trying to get tokens from users in a group and parse result with poweshell. I've tried to use the command :

gam.exe group _groupID_  show tokens
but is hard to parse the result, so I found that I can run:

gam.exe group _groupID_  print tokens

So, i get a CSV that can easily parse with powershell.  The problem is that the first command return Client ID field, but the second is empty. 

Full steps to reproduce the issue:
1. gam.exe group _<group>_  show tokens
2. print 
 User: lucas@domain.com.ar, Access Tokens (1/1)
  Client ID: XXXXXXX.apps.googleusercontent.com
  ......
3.gam.exe group _<group>_  print tokens
4. print
user,clientId,displayText,anonymous,nativeApp,userKey,scopes
lucas@domain.com.ar,,Google APIs Explorer,False,False,118102598514375220671

**(Note two comas between my email and "google APIs Explorer")**

Expected outcome (what are you trying to do?):
user,clientId,displayText,anonymous,nativeApp,userKey,scopes
lucas@domain.com.ar,**XXXXXXX.apps.googleusercontent.com**,Google APIs Explorer,False,False,118102598514375220671

Any clue ?

**Comment by lucasndi** [comment]
I solve this. 
Line 8255 of gam.py should be : 

`  row = {u'user': user, u'clientId': u' '+**token[u'clientId']**, u'scopes': u' '.join(token.get(u'scopes', []))}`

**Comment by jay0lee** [comment]
Fixed by #308 and GAM 3.8.
